### PR TITLE
Fix layout issues

### PR DESCRIPTION
**Pull request by alexilyaev**

### CODE REVIEW

File: lib/ui/assets/stylesheets/_buttons.scss [hunk 1]
```diff
@@ -4,20 +4,46 @@
 
 .btn {
   background: $colorGray;
-  display: inline-block;
+  border: 0;
+  border: 0;
```
Comment (ilyagelman) [inline lib/ui/assets/stylesheets/_buttons.scss]: duplicate

File: lib/ui/components/active-indicator.js [hunk 2]
```diff
@@ -11,7 +11,11 @@ export class ActiveIndicator extends React.Component {
   }
 
   _indicatorStyle() {
-    return cx('active-indicator', { active: this.props.active });
+    return cx(
+      'active-indicator',
+      { active: this.props.active },
+      { invisible: this.props.showOnlyActive && !this.props.active }
```
Comment (ilyagelman) [inline lib/ui/components/active-indicator.js]: Why two objects instead of one?

File: lib/ui/components/captured-requests.js [hunk 3]
```diff
@@ -48,15 +49,18 @@ export class CapturedRequests extends React.Component {
...
 
-      return mockedRequest.states
+      mockedState = mockedRequest.states
         .filter((state) => state.id === mockedRequest.selectedStateId)[0]
         .name;
     }
 
-    return <Url url={ request.url }/>;
+    return <Url url={ request.url }
+                mockedState={ mockedState }/>;
```
Comment (ilyagelman) [inline lib/ui/components/captured-requests.js]: mockedState should not be a part of url component, it is only for url string presentation (truncation and colored parameters)

File: lib/ui/components/captured-requests.js [hunk 4]
```diff
@@ -94,15 +93,19 @@ export class CapturedRequests extends React.Component {
...
-             className={ this._requestRowClassName(request.id) }
              onClick={ this._selectRequest.bind(this, request)}>
-          <div>
-            { this._activeIndicator(request) }
-            <MethodLabel method={ request.method }/>
-            { this._requestDetailsFor(request) }
-          </div>
-
-          { this._actionButtonFor(request) }
+          <RequestRow requestId={ request.id }
```
Comment (ilyagelman) [inline lib/ui/components/captured-requests.js]: Please take into notice that RequestRow component is draggable to scenario name

File: lib/ui/components/captured-requests.js [hunk 5]
```diff
@@ -94,15 +93,19 @@ export class CapturedRequests extends React.Component {
...
+                      selected={ this._isSelected(request.id) }>
+            <div className="request-info">
+              <ActiveIndicator active={ this._isAlreadyMocked(request) }
+                               onToggle={ () => {} }
+                               showOnlyActive={ true }/>
```
Comment (ilyagelman) [inline lib/ui/components/captured-requests.js]: If you add props to components, make sure to update propTypes

File: lib/ui/components/scenario-name.js [hunk 6]
```diff
@@ -59,10 +59,13 @@ export class ScenarioName extends React.Component {
       <li className="scenario-name" onClick={ this._selectScenario.bind(this) }>
 
         <a className={ this._rowClassName() }>
-          <ActiveIndicator active={ API.currentScenario === this.props.scenario.id }
-                           onToggle={ () => this._toggleCurrentScenario() }/>
+          <div>
+            <ActiveIndicator active={ API.currentScenario === this.props.scenario.id }
+                             onToggle={ () => this._toggleCurrentScenario() }
+                             showOnlyActive={ true }/>
```
Comment (ilyagelman) [inline lib/ui/components/scenario-name.js]: Why showOnlyActive here?

File: lib/ui/assets/stylesheets/_buttons.scss (hunk 1 above)
Comment (alexilyaev) [inline lib/ui/assets/stylesheets/_buttons.scss]: Done

File: lib/ui/components/active-indicator.js (hunk 2 above)
Comment (alexilyaev) [inline lib/ui/components/active-indicator.js]: Yeah, forgot it works this way

File: lib/ui/components/captured-requests.js (hunk 4 above)
Comment (alexilyaev) [inline lib/ui/components/captured-requests.js]: We decided it makes sense in it's as a component, e.g. selected row etc.
So we should move the Draggable outside the component or something.

File: lib/ui/components/captured-requests.js (hunk 5 above)
Comment (alexilyaev) [inline lib/ui/components/captured-requests.js]: Done

File: lib/ui/components/scenario-name.js (hunk 6 above)
Comment (alexilyaev) [inline lib/ui/components/scenario-name.js]: You're right, fixed

File: lib/ui/components/captured-requests.js (hunk 3 above)
Comment (alexilyaev) [inline lib/ui/components/captured-requests.js]: Reverting this

**Pull request merged**